From afa902147637065f9c893879a1d06e0320c302d0 Mon Sep 17 00:00:00 2001
From: Gan Yi Fang <yi.fang.gan@intel.com>
Date: Thu, 22 Sep 2022 05:43:42 -0400
Subject: [PATCH 5/5] ethtool: Fix the enum message type

This patch fix the enum message type for both way communication between
kernel and userspace.

Signed-off-by: Gan Yi Fang <yi.fang.gan@intel.com>
---
 uapi/linux/ethtool_netlink.h | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/uapi/linux/ethtool_netlink.h b/uapi/linux/ethtool_netlink.h
index 438a03c..ce10d6d 100644
--- a/uapi/linux/ethtool_netlink.h
+++ b/uapi/linux/ethtool_netlink.h
@@ -47,9 +47,11 @@ enum {
 	ETHTOOL_MSG_MODULE_EEPROM_GET,
 	ETHTOOL_MSG_STATS_GET,
 	ETHTOOL_MSG_PHC_VCLOCKS_GET,
+	ETHTOOL_MSG_MODULE_GET,
+	ETHTOOL_MSG_MODULE_SET,
 	ETHTOOL_MSG_PREEMPT_GET,
 	ETHTOOL_MSG_PREEMPT_SET,
-
+	
 	/* add new constants above here */
 	__ETHTOOL_MSG_USER_CNT,
 	ETHTOOL_MSG_USER_MAX = __ETHTOOL_MSG_USER_CNT - 1
@@ -92,6 +94,8 @@ enum {
 	ETHTOOL_MSG_MODULE_EEPROM_GET_REPLY,
 	ETHTOOL_MSG_STATS_GET_REPLY,
 	ETHTOOL_MSG_PHC_VCLOCKS_GET_REPLY,
+	ETHTOOL_MSG_MODULE_GET_REPLY,
+	ETHTOOL_MSG_MODULE_NTF,
 	ETHTOOL_MSG_PREEMPT_GET_REPLY,
 	ETHTOOL_MSG_PREEMPT_NTF,
 
-- 
2.34.1

